index.js

```diff
@@ -11,63 +12,86 @@ const QUESTIONS = [{
         input: "[1,2,3,4,5]",
         output: "5"
     }]
-}];
+}]
 
 
 const SUBMISSION = [
 
 ]
 
-app.post('/signup', function(req, res) {
-  // Add logic to decode body
-  // body should have email and password
 
+app.post('/signup', (req, res)=>{
+    // Add logic to decode body
+    // body should have email and password
+    //check if user already exists
+    //Store email and password (as is for now) in the USERS array above (only if the user with the given email doesnt exist)
+    // return back 200 status code to the client
 
-  //Store email and password (as is for now) in the USERS array above (only if the user with the given email doesnt exist)
```

Comment:
It's always better to decode the request body before trying to use it in your functions.

this lets you end the API call upfront in case of bad request